src/components/ArchiveProject/index.tsx

```diff
@@ -38,6 +40,28 @@ export default function ArchiveProject({
 
   const [isLoadingArchive, setIsLoadingArchive] = useState<boolean>(false)
 
+  const revalProject = async () => {
```

Comment:
Perhaps?

```suggestion
  const revalidateProjectAfterArchive = async () => {
```

Or

```suggestion
  const cvAwareRevalidateProject = async () => {
```

idk lol